src/components/TagMenu/index.tsx

```diff
@@ -4,7 +4,6 @@ import {msg, Trans} from '@lingui/macro'
 import {useLingui} from '@lingui/react'
 import {useNavigation} from '@react-navigation/native'
 
-import {makeSearchLink} from '#/lib/routes/links'
```

Comment:
You can delete this util now I suppose?

Reply:
We may need it again in the future, might as well leave it imo